### PR TITLE
Add columns to household_member

### LINKED ISSUE #172: Add columns to Household_Member table
Looks like we need:

- Literacy (as well as education level)
- BMI

Remember to update the format specifications.

Related to issue #142

### LINKED ISSUE #173: Survey Dates - Month Only?
Can we store survey dates i.e. dates related to the survey as month and year only? That is how they are arriving in the data and we have a steer from Fanny that its as much precision as is required.

Related to issue #142

### LINKED ISSUE #174: Add Region to Household table
Its an attribute coming through in the data. Needs adding to the table.

Related to issue #142

### PR DESCRIPTION
**Pull request by andy-bevan**
Add literacy, weight, height, bmi (calculated from weight and height) to household member
And region to household.
Change survey date types to text

Closes #174 
Closes #172 
Closes #173

### CODE REVIEW

File: db/objects/V0.1.0.500__Household_member.sql [hunk 1]
```diff
@@ -3,7 +3,7 @@ CREATE TABLE household_member (
     , household_id         bigint  references household (id)
     , original_id          text
     , sex                  text CHECK (sex in ('m','f'))
-    , education_level      text CHECK (education_level IN ('none', 'primary', 'secondary', 'tertiary'))
+    , education_level      text CHECK (education_level IN ('none', 'primary', 'secondary', 'tertiary','higher'))
```
Comment (rbroth) [inline db/objects/V0.1.0.500__Household_member.sql:6]: I don't think we want 'higher'; it's a synonym for 'tertiary'. Still waiting for @fannymawbey confirming on https://github.com/micronutrientsupport/database-architecture/issues/160

File: db/objects/V0.1.0.500__Household_member.sql [hunk 2]
```diff
@@ -32,4 +37,9 @@ COMMENT on COLUMN household_member.had_diarrhoea    IS 'Whether this person has
 COMMENT on COLUMN household_member.survey_cluster   IS 'The surveying cluster used for selecting this individual'; -- TODO: more info
 COMMENT on COLUMN household_member.survey_strata    IS 'The surveying strata used for selecting this individual';  -- TODO: more info
 COMMENT on COLUMN household_member.survey_weight    IS 'The surveying weight used for selecting this individual';  -- TODO: more info
+COMMENT on COLUMN household_member.literacy         IS 'The level of literacy the person has attained';
+COMMENT on COLUMN household_member.weight_in_kg     IS 'The weight of the person in kilograms';
+COMMENT on COLUMN household_member.height_in_cm     IS 'The height of the person in centimetres';
+COMMENT on COLUMN household_member.bmi				IS 'The body mass index of the person calculated from weight and height as weight in kg/height in metres squared';
```
Comment (rbroth) [inline db/objects/V0.1.0.500__Household_member.sql:43]: whitespace

File: db/objects/V0.1.0.500__Household_member.sql (hunk 1 above)
Comment (andy-bevan) [reply db/objects/V0.1.0.500__Household_member.sql:6]: I've commented in the issue.

File: db/objects/V0.1.0.500__Household_member.sql (hunk 2 above)
Comment (andy-bevan) [reply db/objects/V0.1.0.500__Household_member.sql:43]: I'll fix that.

**Comment by rbroth** [comment]
Looks good, apart from the `higher` education bit. I'd prefer to get that sorted, because as it stands at the moment it just feels like we're chucking in data without understanding why we're storing it

**Comment by andy-bevan** [comment]
> Looks good, apart from the `higher` education bit. I'd prefer to get that sorted, because as it stands at the moment it just feels like we're chucking in data without understanding why we're storing it

I see what you mean. But the interview designers must have thought that 5 levels were required so I imagine they would expect to search or filter on 5 levels when they interrogate the data.  Tertiary and university are not really the same thing if tertiary means college/a levels/vocational training...

**Comment by rbroth** [comment]
> 
> 
> > Looks good, apart from the `higher` education bit. I'd prefer to get that sorted, because as it stands at the moment it just feels like we're chucking in data without understanding why we're storing it
> 
> I see what you mean. But the interview designers must have thought that 5 levels were required so I imagine they would expect to search or filter on 5 levels when they interrogate the data. Tertiary and university are not really the same thing if tertiary means college/a levels/vocational training...

Yes, but the question is: is the data design of this particular survey consistent with what we're trying to achieve in our system? I don't feel confident that the answer is "yes", since we don't even know the definitions of those terms as used by the survey

File: db/objects/V0.1.0.500__Household_member.sql (hunk 1 above)
Comment (rbroth) [reply db/objects/V0.1.0.500__Household_member.sql:6]: decision at standup: put it in

**Review by rbroth — APPROVED** [review]

**Pull request merged**